install/silverbullet-install.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2024 community-scripts ORG
+# Author: Dominik Siebel (dsiebel)
+# License: MIT
+# https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+
+source /dev/stdin <<< "$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies"
+$STD apt-get install -y curl
+$STD apt-get install -y sudo
+$STD apt-get install -y mc
+msg_ok "Installed Dependencies"
+
+RELEASE=$(curl -s https://api.github.com/repos/silverbulletmd/silverbullet/releases/latest | grep "tag_name" | awk '{print substr($2, 2, length($2)-3) }')
+
+msg_info "Installing ${APPLICATION}"
```

Comment:
App

Reply:
`$APP` is only available in update script, if I am not mistaken.
In install scripts it needs to be `$APPLICATION`.

Reply:
Double checked with the scripts you mentioned, to be super-sure:

- kimai: `APPLICATION`
- komga: `APPLICATION`
- snipeit: none, hardcoded `snipe-it` or `snipeit`
- vikunia: `APPLICATION`
- bookstack: both `APPLICATION` and hard-coded `bookstack`